spring-integration-redis/src/test/java/org/springframework/integration/redis/outbound/RedisOutboundGatewayTests.java

```diff
@@ -161,7 +161,7 @@ public void testMGetCommand() {
 		byte[] value2 = "bar2".getBytes();
 		connection.set("foo1".getBytes(), value1);
 		connection.set("foo2".getBytes(), value2);
-		this.mgetCommandChannel.send(MessageBuilder.withPayload(new String [] {"foo1", "foo2"}).build());
+		this.mgetCommandChannel.send(MessageBuilder.withPayload(new String[] {"foo1", "foo2"}).build());
```

Comment:
Yes; GH is flaky - in fact there were 2 more commits that only just appeared in the PR - they were pushed 30 minutes ago. See below.